pkg/apis/kops/validation/validation.go

```diff
@@ -367,6 +367,7 @@ func validateNetworking(c *kops.ClusterSpec, v *kops.NetworkingSpec, fldPath *fi
 	}
 
 	if v.Romana != nil {
+		allErrs = append(allErrs, field.Forbidden(fldPath.Child("romana"), "support for Romana has been removed"))
 		if optionTaken {
```

Comment:
This if block can be removed

Reply:
Can it? If I remove that block, kops creates my cluster.

Reply:
A validation error should prevent creation. How is that happening?

Reply:
Sorry, I thought you were referring to the outer if. Yeah, this one can technically be removed.